### ISSUE TITLE
Wrong postionAbsolute when creating new node with parentNode parameter

### ISSUE THREAD

**Issue by WexLab** [wbkd/react-flow#2433]
### Describe the Bug

When I try to create new nodes with parentNode parameter, their positionAbsolute parameter is not the same as position parameter.

### Your Example Website or App

https://reactflow.dev/docs/examples/interaction/drag-and-drop/

### Steps to Reproduce the Bug or Issue

Just use DND example https://reactflow.dev/docs/examples/interaction/drag-and-drop/

```
const newNode = {
        id: getId(),
        type,
        position,
        data: { label: `${type} node` },
        parentNode: '1' // add this string
};
```

### Expected behavior

The new node is supposed to appear next to the cursor, this bug is not limited to the DND example.

### Platform

all

**Comment by bcakmakoglu** [comment]
I think you're misunderstanding what `positionAbsolute` does.
When a node is set as the child of another node, the `position` you set is relative to it's parent node, meaning `{ x: 0, y: 0 }` on a child node would put it to the top left corner of it's parent, node the view pane itself.
`positionAbsolute` on the other hand is used, as the name suggests, to apply the *absolute* position of that node as it's transform values - in general it's not a value you should be working with or attempt to modify as it's an internal property.

You can test this in the DnD example by simply using `{ x: 0, y: 0 }` for new nodes you drop in, you will see (if you set the `parentNode` property as well of course) that the new nodes will lay on top of the parent.

So in conclusion: This is not a bug, it's working exactly as intended.

**Comment by moklick** [comment]
Thanks @bcakmakoglu !

@WexLab as @bcakmakoglu pointed out everything works as expected. If you want to add a node to a subflow by drag and drop, you would need to subtract the parent position before you add the new node since the `position` attribute is always relative to its parent.
